include/RAJA/policy/openmp/reduce.hpp

```diff
@@ -102,7 +102,7 @@ class ReduceOMP
 
 } /* detail */
 
-RAJA_DECLARE_ALL_REDUCERS(omp_reduce, detail::ReduceOMP);
+RAJA_DECLARE_ALL_REDUCERS(omp_reduce, detail::ReduceOMP)
```

Comment:
Should these be in this PR?

Reply:
We can break them out, but they came from the same warning checking report from @tepperly.  It's only a warning with -Werror=pedantic apparently, but it is a warning nevertheless.

Reply:
Okay, fine with me since it's a minor change. 

Perhaps we should add `-Werror=pedantic` to our flags for Travis....

Reply:
Maybe, I'm not sure what all it will pick up, but if we're already clean we may as well stay that way.

Reply:
Exactly 😄